### ISSUE TITLE
Accessibility errors found in your template

### ISSUE THREAD

**Issue by wapyce** [chrisbobbe/jekyll-theme-prologue#40]
The user @carlsonsantana validate your site template "https://chrisbobbe.github.io/jekyll-theme-prologue/" and found these accessibility errors:
* Page https://chrisbobbe.github.io/jekyll-theme-prologue/:
  * Issue *7e39b0a8-fd58-4b62-88ba-650cc302d7ad*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
    * **Message**: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
    * **Selector**: `#contact > div > form > div > div:nth-child(3) > textarea`
    * **Context**: `<textarea name="message" placeholder="Message"></textarea>`
  * Issue *f97dc67c-cc6c-4dcc-84b3-c2f6feadb588*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Textarea.Name
    * **Message**: This textarea element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.
    * **Selector**: `#contact > div > form > div > div:nth-child(3) > textarea`
    * **Context**: `<textarea name="message" placeholder="Message"></textarea>`
  * Issue *e07ebf9e-1a4c-40e6-b717-5a7bd4df8565*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
    * **Message**: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
    * **Selector**: `#contact > div > form > div > div:nth-child(2) > input`
    * **Context**: `<input type="text" name="email" placeholder="Email">`
  * Issue *e7d6b30e-6744-4a6e-9a0f-2ae37c20370b*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputText.Name
    * **Message**: This textinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.
    * **Selector**: `#contact > div > form > div > div:nth-child(2) > input`
    * **Context**: `<input type="text" name="email" placeholder="Email">`
  * Issue *c9d1239f-0818-498e-9000-12acfa1b4152*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
    * **Message**: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
    * **Selector**: `#contact > div > form > div > div:nth-child(1) > input`
    * **Context**: `<input type="text" name="name" placeholder="Name">`
  * Issue *7b950923-2755-453c-b4e4-bb411e981c61*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputText.Name
    * **Message**: This textinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.
    * **Selector**: `#contact > div > form > div > div:nth-child(1) > input`
    * **Context**: `<input type="text" name="name" placeholder="Name">`
  * Issue *25c73ead-bcfb-4689-927a-ecdda544f908*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_1.1_1_1.H30.2
    * **Message**: Img element is the only content of the link, but is missing alt text. The alt text should describe the purpose of the link.
    * **Selector**: `#about-me > div > a`
    * **Context**: `<a href="#" class="image featured"><img src="assets/images/pic08.j...</a>`
  * Issue *fbe3fc6c-8481-4c80-b2d9-1187597d1e87*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2
    * **Message**: The html element should have a lang or xml:lang attribute which describes the language of the document.
    * **Selector**: `html`
    * **Context**: `<html><head>
  <meta n...</html>`

You can check these accessibility errors using [pa11y](https://github.com/pa11y/pa11y).
You can view the [full validation results in our website](https://wapyce.herokuapp.com/validation/validations/9c57f2b2-e530-4cfe-8e9c-ff507a2262e6/).
